README.md

```diff
@@ -1,7 +1,128 @@
+KerasCV can be understood as a horizontal extension of the Keras API: they're new first-party Keras 
+objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive 
+the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that 
+are maintained by the Keras team itself (unlike TFAddons).
 
-## Under Construction
-KerasCV is a relatively new project.  Expect regular updates and changes in the near future.
+# No Goals
+- **KerasCV is not a repository of blackbox end-to-end solutions (like TF Hub or ModelGarden).**
+
+    For the time being, it is focused on modular and reusable building blocks. We expect models in 
+    ModelGarden or TF Hub to eventually be built on top of the building blocks provided by KerasCV.
+
+    In the process of developing these building blocks, we will by necessity implement end-to-end 
+    workflows, but they're intended purely for demonstration and grounding purposes, they're not 
+    our main deliverable.
+
+
+- **KerasCV is not a repository of low-level image-processing ops, like tf.vision.** 
+
+    KerasCV is fundamentally an extension of the Keras API: it hosts Keras objects, like layers, 
+    metrics, or callbacks. Low-level C++ ops should go elsewhere.
+
+
+- **KerasCV is not a research library.**
+
+    Researchers may use it, but we do not consider researchers to be our target audience. Our target 
+    audience is applied CV engineers with experimentation and production needs. KerasCV should make 
+    it possible to quickly reimplement industry-strength versions of the latest generation of 
+    architectures produced by researchers, but we don't expect the research effort itself to be built
+    on top of KerasCV. This enables us to focus on usability and API standardization, and produce 
+    objects that have a longer lifespan than the average research project.
+
+## Areas of focus
+At the start of the project, we would like to take a task-oriented approach to outline our project, 
+and the first 2 tasks we would like to focus are:
+
+1. **Image classification** (Given an image and a list of labels, return the label and confidence for 
+   the image).
+2. **Object detection** (Given an image and a list of labels, return a list of bounding box, 
+   corresponding labels and confidence for the objects in the image).
+
+For each of the tasks, we are targeting to provide user model architectures, layers, preprocessing
+functions, losses, metrics and other necessary components to achieve state of the art accuracy.
+
+There will be common shareable components between those 2 tasks, like model building blocks, etc. We
+will have them in the repository as public API as well.
+
+### Common components
```

Comment:
I suppose this is derived by the original proposal right?

Reply:
We will move this to the roadmap and not included in the readme.

Reply:
Yes it is better. Also pointing on a github Wiki page it could be ok or easier to update.